noir_stdlib/src/meta/expr.nr

```diff
@@ -329,6 +346,15 @@ fn new_array(exprs: [Expr]) -> Expr {
     quote { [$exprs]}.as_expr().unwrap()
 }
 
+fn new_assert(predicate: Expr, msg: Option<Expr>) -> Expr {
+    if msg.is_some() {
+        let msg = msg.unwrap();
```

Comment:
```suggestion
    if let Some(msg) = msg {
```

Just some syntax sugar.

Reply:
Ah, this is Noir 😊 

(I swear I wrote `Some(...)` a few times while coding in Noir too)